import not working in NextJS app

first of all, super happy i found this repo! i was going to implement the basics of this myself, but y'all seem to already have made a great job!

when importing the lib in a NextJS app i get the below error:
```
Error: Cannot find module './renderer-registry-entries'
Require stack:
- /Users/fant/repos/ai/petridish/lab/.next/server/vendor-chunks/@great-expectations.js
- /Users/fant/repos/ai/petridish/lab/.next/server/webpack-runtime.js
- /Users/fant/repos/ai/petridish/lab/.next/server/app/petridish/[petridishId]/v/[versionId]/[view]/page.js
    at __webpack_require__ (/Users/fant/repos/ai/petridish/lab/.next/server/webpack-runtime.js:33:43)
    at eval (./src/components/petridish/trace/TraceList.tsx:24:102)
    at (ssr)/./src/components/petridish/trace/TraceList.tsx (/Users/fant/repos/ai/petridish/lab/.next/server/app/petridish/[petridishId]/v/[versionId]/[view]/page.js:345:1)
    at Object.__webpack_require__ [as require] (/Users/fant/repos/ai/petridish/lab/.next/server/webpack-runtime.js:33:43)
digest: "1312251077"
 GET /petridish/7bf37335-8787-59ca-95a2-061dfafb66ff/v/e9acb592-6b0d-4612-8e15-d25b0c00728f/chat 500 in 6977ms
 ⚠ ./src/components/petridish/trace/TraceList.tsx
Attempted import error: 'rendererRegistryEntries' is not exported from '@great-expectations/jsonforms-antd-renderers' (imported as 'rendererRegistryEntries').

Import trace for requested module:
./src/components/petridish/trace/TraceList.tsx

./src/components/petridish/trace/TraceList.tsx
Attempted import error: 'cellRegistryEntries' is not exported from '@great-expectations/jsonforms-antd-renderers' (imported as 'cellRegistryEntries').

Import trace for requested module:
./src/components/petridish/trace/TraceList.tsx
```


when I require it, it works as expected. `renderer-registry-entries.js` exists in lib/esm, where `index.mjs` imports it, but for some reason it's not working. lmk if i can help debug or provide any other information to track down the issue!

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
     "lib/**/*"
   ],
   "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
   "types": "./lib/cjs/index.d.ts",
   "exports": {
     "require": {
@@ -24,7 +25,7 @@
     },
     "import": {
       "types": "./lib/esm/index.d.ts",
-      "default": "./lib/esm/index.mjs"
+      "default": "./lib/esm/index.js"
     },
     "default": "./lib/cjs/index.js"
   },
@@ -44,7 +45,7 @@
   "scripts": {
     "clean": "rm -rf ./lib",
     "build": "pnpm run clean && pnpm run build:esm && pnpm run build:cjs",
-    "build:esm": "tsc -p tsconfig.esm.json && mv lib/esm/index.js lib/esm/index.mjs",
+    "build:esm": "tsc -p tsconfig.esm.json",
     "build:cjs": "tsc -p tsconfig.cjs.json",
     "prepack": "pnpm run build",
     "test": "vitest",

